testdata/p4_16_samples_outputs/psa-basic-counter-bmv2.p4.spec

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
I am guessing that this is the reference output produced by the compiler.
Is this checked by some script?

Reply:
This is used as input to dpdk datapath. Currently not checked by any script in this repo, but it could be used drive a real test in the future. At the moment, this is used to keep track of the compiler output, making sure no accidental change to the output.